connectors-e2e-test/connectors-e2e-test-servicenow/pom.xml

```diff
@@ -0,0 +1,44 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>io.camunda.connector</groupId>
+    <artifactId>connectors-e2e-test-parent</artifactId>
+    <version>8.8.0-SNAPSHOT</version>
+  </parent>
+  <artifactId>connectors-e2e-test-servicenow</artifactId>
+  <name>Archetype - connectors-e2e-test-servicenow</name>
+  <url>http://maven.apache.org</url>
+  <dependencies>
+<!--    <dependency>-->
+<!--      <groupId>io.camunda</groupId>-->
+<!--      <artifactId>camunda-process-test-spring</artifactId>-->
+<!--    </dependency>-->
```

Comment:
Shouldnt CPT be used when running this on the CI?

Reply:
from my understanding, the connector runtime is part of CPT, running in the isolated testcontainer environment - and thus, can not reach out (e.g. via the `REST` connector) to external systems. Which is what we'd need in order to connect to the live SN instance.

Reply:
now works with CPT and bootstrapped Connectors runtime